pkg/gencmd/command.go

```diff
@@ -0,0 +1,77 @@
+package gencmd
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type CommandType int
+
+const (
+	CommandUnknown CommandType = iota
+	CommandGenerate
+	CommandDestroy
+)
+
+type Command struct {
+	Use             string
+	Short           string
+	Long            string
+	Example         string
+	Args            cobra.PositionalArgs
+	BuildParams     func(c *Command, args []string) (interface{}, error)
+	PostRun         func(c *Command, args []string) error
+	ShouldRun       ShouldRunFunc
+	ShouldInsideApp bool
+	TemplateFS      http.FileSystem
+
+	flags *pflag.FlagSet
+	ctx   *Ctx
+}
+
+func (c *Command) Flags() *pflag.FlagSet {
+	if c.flags == nil {
+		c.flags = new(pflag.FlagSet)
+	}
+	return c.flags
+}
+
+func (c *Command) Ctx() *Ctx {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Command.Ctx should have comment or be unexported